Adding Import to the Export pipeline docs (#36090)
DataDog/documentation: content/en/observability_pipelines/configuration/export_and_import_pipeline_configurations.md

---

> title: Export and Import a Pipeline Configuration in JSON or Terraform

Reviewer: ```suggestion
title: Export and Import a Pipeline Configuration in JSON or Terraform
aliases:
  - /observability_pipelines/configuration/export_pipeline_configuration/
```

> title: Export and Import a Pipeline Configuration in JSON or Terraform

Reviewer: Current title implies both formats apply to both directions, but import is JSON-only
```suggestion
title: Export and Import a Pipeline Configuration
```

   - [API documentation][5]
   - [Terraform documentation][4]

> ### Import a pipeline's JSON configuration to the UI

Reviewer: Promote to h2 or rename the parent h2 to include "import"
```suggestion
## Import a pipeline's JSON configuration to the UI
```

> To help build your pipelines and easily transfer pipeline configurations across Datadog orgs, you can also import a pipeline's JSON to the UI.

Reviewer: ```suggestion
To help build your pipelines and transfer pipeline configurations across Datadog orgs, you can also import a pipeline's JSON to the UI.
```

1. Navigate to [Observability Pipelines][1].
1. Select a draft pipeline to view the configuration.
1. Click on the settings and select **Import Pipeline**
   {{< img src="observability_pipelines/export_pipeline/import_access.png" alt="The settings menu showing Import Pipeline" style="width:100%;" >}}

> 1. Choose a pipeline configuration as a JSON file from your local storage.

Reviewer: ```suggestion
1. Choose a pipeline configuration as a JSON file from your computer.
```

   {{< img src="observability_pipelines/export_pipeline/import_select.png" alt="The import modal giving the option to select a JSON file" style="width:100%;" >}}

> 1. Click Import. This will replace your current pipeline configuration with the imported JSON's pipeline configuration. 

Reviewer: ```suggestion
1. Click Import. This replaces your current pipeline configuration with the imported JSON's pipeline configuration. 
```

> 1. [Install the Observability Pipelines Worker and deploy your pipeline using the UI][8]

Reviewer: ```suggestion
1. [Install the Observability Pipelines Worker and deploy your pipeline using the UI][8].
```
